Anchor.toml

```diff
@@ -3,7 +3,7 @@ url = "https://anchor.projectserum.com"
 
 [provider]
 cluster = "localnet"
-wallet = "/home/armaniferrante/.config/solana/id.json"
+wallet = "~/.config/solana/id.json"
 
 [scripts]
-test = "ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts"
+test = "npx ts-mocha -t 1000000 tests/**/*.ts"
```

Comment:
even though I don't haave ts-mocha globally, for some reason using npx makes it work a bit more reliably for me on different machines. I get errors that the tests/ dir isn't found sometimes otherwise.

I don't think specifying tsconfig here is necessary either but can happily revert these change.

Reply:
this is better